kinova_gen3_7dof/gen3_7dof.xml

```diff
@@ -0,0 +1,124 @@
+<mujoco model="gen3">
+  <compiler angle="radian" meshdir="assets"/>
+
+  <option integrator="implicitfast"/>
+
+  <default>
+    <!-- kinova defaults -->
+    <default class="kinova">
+      <material specular="0.5" shininess="0.25"/>
+      <joint armature="0.1" damping="1" axis="0 0 1" range="-2.8973 2.8973"/>
+      <general dyntype="none" biastype="affine" ctrlrange="-6.28 6.28" forcerange="-87 87"/>
+      <default class="infiniteJoints">
+        <joint limited="false"/>
+      </default>
+      <default class="limitedJoints">
+        <joint limited="true"/>
+      </default>
+      <site size="0.001" rgba="0.5 0.5 0.5 0.3" group="4"/>
+
+      <default class="visual">
+        <geom type="mesh" contype="0" conaffinity="0" group="2"/>
+      </default>
+      <default class="collision">
+        <geom type="mesh" group="3" condim="6"/>
```

Comment:
are we sure we want `condim=6`?

Reply:
https://github.com/PickNikRobotics/mujoco_menagerie/pull/1/commits/ae6d82ebd0951554d58229c51831c7b024165f0f